app/models/concerns/pxe_loader_validator.rb

```diff
@@ -0,0 +1,14 @@
+module PxeLoaderValidator
+  extend ActiveSupport::Concern
+
+  included do
+    validate :validate_pxe_loader
+  end
+
+  def validate_pxe_loader
+    if operatingsystem && pxe_loader.present?
```

Comment:
`return unless operatingsystem && pxe_loader.present?` to avoid making the method an if condition. I'll enable the cop later - http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/GuardClause